source/documentation/administration_guide/topics/proc_migrating_she_host_to_different_cluster.adoc

```diff
@@ -0,0 +1,30 @@
+// module included:
+// doc-Administration_Guide/chap-Hosts.adoc
+
+[id='migrating_she_host_to_different_cluster_{context}']
+= Migrating a self-hosted engine host to a different cluster
+
+You cannot migrate a host that is configured as a self-hosted engine host to a data center or cluster other than the one in which the self-hosted engine virtual machine is running. All self-hosted engine hosts must be in the same data center and cluster.
+
+You need to disable the host from being a self-hosted engine host by undeploying the self-hosted engine.
+
+.Procedure
+
+. Click menu:Compute[Hosts] and select the host.
```

Comment:
Suggestion: Start this sentence with the location unless it is obvious (For example, "In the XYZ manager, ..."

Reply:
Yeah, I thought about that. It should be understood that you're already in the Administration Portal.

I remember that was something that was not clear to me when I onboarded.

Reply:
OK, in that case, go with your original. I think there may be a global issue here, but this PR is not the place to address it.